cpp/README.md

```diff
@@ -279,7 +279,83 @@ both of these options would be used rarely. Current known uses-cases when they a
 
 *  Parameterized tests in google test.
 
+## Nightly Builds of `arrow-cpp`, `parquet-cpp`, and `pyarrow` for Linux
+
+Nightly builds of conda packages are available for [`arrow-cpp`][5],
+[`parquet-cpp`][6], and [`pyarrow`][7].
```

Comment:
We aren't allowed to advertise these in the project per ASF policies https://issues.apache.org/jira/browse/ARROW-1935

Reply:
Cool, will remove.